src/playground/player.jsx

```diff
@@ -28,6 +28,7 @@ const Player = ({isPlayerOnly, onSeeInside}) => (
         <GUI
             enableCommunity
             isPlayerOnly={isPlayerOnly}
+            {...props}
```

Comment:
is there a specific prop this was missing? Curious if you really want to kind of steal the params list from GUI here, or just list the ones you care about, or if it wouldn't make more sense to pass GUI in as a child?

Reply:
Good idea, I updated to use an explicit prop.